### PR TITLE
gh-119174: Fix high DPI causes turtledemo(turtle-graphics examples) windows blurry

### PR DESCRIPTION
**Pull request by Wulian233**
* Issue: gh-119174

Copy `https://github.com/python/cpython/blob/main/Lib/idlelib/pyshell.py#L16-L22` to `https://github.com/python/cpython/blob/main/Lib/turtledemo/__main__.py`

``` python
if win32:
    try:
        import ctypes
        PROCESS_SYSTEM_DPI_AWARE = 1  # Int required.
        ctypes.OleDLL('shcore').SetProcessDpiAwareness(PROCESS_SYSTEM_DPI_AWARE)
    except (ImportError, AttributeError, OSError):
        pass
```

### CODE REVIEW

File: Lib/turtledemo/__main__.py [hunk 1]
```diff
@@ -96,8 +96,10 @@
 
 import turtle
 
+
```
Comment (nineteendo) [inline Lib/turtledemo/__main__.py:101]: Remove this line:

```suggestion
```

Comment (terryjreedy) [reply Lib/turtledemo/__main__.py:101]: Correct.  Double space only recommended for function or class definition.

File: Misc/NEWS.d/next/Library/2024-05-19-18-49-04.gh-issue-119174.5GTv7d.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Fix high DPI causes turtledemo(turtle-graphics examples) windows blurry
+Patch by Wulian233
```
Comment (terryjreedy) [inline Misc/NEWS.d/next/Library/2024-05-19-18-49-04.gh-issue-119174.5GTv7d.rst]:
```suggestion
Patch by Wulian233

```

File: Lib/turtledemo/__main__.py (hunk 1 above)
Comment (nineteendo) [reply Lib/turtledemo/__main__.py:101]: Well, it's mostly because we shouldn't make stylistic changes to existing code until pep 8 is enforced.

**Comment by terryjreedy** [comment]
Lint failed because news item needed newline.  I should have replaced default "Update ..." message as change is not visible in diffs.

I will make unrelated config.py correction in a separate PR.

I will change this to move fixup code to a new function in idlelib.util and then conditionally import and call function.

**Comment by Wulian233** [comment]
Understand... so do I need to revert the changes to config.py in this PR?

**Comment by terryjreedy** [comment]
Wait for now.

File: Misc/NEWS.d/next/Library/2024-05-19-18-49-04.gh-issue-119174.5GTv7d.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
```
Comment (terryjreedy) [inline Misc/NEWS.d/next/Library/2024-05-19-18-49-04.gh-issue-119174.5GTv7d.rst]:
```suggestion
Patch by Wulian233 and Terry Jan Reedy

```

**Review by terryjreedy — CHANGES_REQUESTED** [review]

**Comment by terryjreedy** [comment]
Ignore change request; I made all changes I think are needed.  After separate config patch, I had to fix merge conflict and decided to make additional change here.  Please pull changes to your system and test that revised code works.

**Comment by Wulian233** [comment]
Thanks! Look Good To Me! 😀

**Review by terryjreedy — APPROVED** [review]

**Pull request merged**